osgp/protocol-adapter-oslp/osgp-adapter-protocol-oslp-elster/src/main/java/org/opensmartgridplatform/adapter/protocol/oslp/elster/infra/messaging/DeviceResponseMessageSender.java

```diff
@@ -84,7 +85,7 @@ private void sendMessage(final ProtocolResponseMessage responseMessage) {
                 responseMessage.getDeviceIdentification(), responseMessage.getMessageType(),
                 responseMessage.getMessagePriority());
 
-        this.oslpResponsesJmsTemplate.send(new MessageCreator() {
+        this.jmsTemplate.send(new MessageCreator() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this anonymous class number of lines from 23 to at most 20, or make it a named class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1188)